.github/CODEOWNERS

```diff
@@ -26,19 +26,22 @@ tests/opentracer                    @DataDog/apm-core-python
 tests/runtime                       @DataDog/apm-core-python
 tests/tracer                        @DataDog/apm-core-python
 
-# CI App
-ddtrace/contrib/pytest              @DataDog/apm-core-python @DataDog/ci-app-libraries
-ddtrace/contrib/pytest_bdd          @DataDog/apm-core-python @DataDog/ci-app-libraries
-ddtrace/contrib/unittest            @DataDog/apm-core-python @DataDog/ci-app-libraries
-tests/contrib/pytest                @DataDog/apm-core-python @DataDog/ci-app-libraries
-tests/contrib/pytest_bdd            @DataDog/apm-core-python @DataDog/ci-app-libraries
-tests/contrib/unittest_plugin       @DataDog/apm-core-python @DataDog/ci-app-libraries
-ddtrace/ext/ci.py                   @DataDog/apm-core-python @DataDog/ci-app-libraries
-ddtrace/ext/test.py                 @DataDog/apm-core-python @DataDog/ci-app-libraries
-ddtrace/internal/codeowners.py      @DataDog/apm-core-python @datadog/ci-app-libraries
-ddtrace/internal/ci_visibility      @DataDog/apm-core-python @DataDog/ci-app-libraries
-tests/ci_visibility                 @DataDog/apm-core-python @DataDog/ci-app-libraries
-tests/tracer/test_ci.py             @DataDog/apm-core-python @DataDog/ci-app-libraries
+# CI App and related
+ddtrace/contrib/coverage            @DataDog/ci-app-libraries
+ddtrace/contrib/pytest              @DataDog/ci-app-libraries
+ddtrace/contrib/pytest_bdd          @DataDog/ci-app-libraries
+ddtrace/contrib/unittest            @DataDog/ci-app-libraries
+tests/contrib/pytest                @DataDog/ci-app-libraries
+tests/contrib/pytest_bdd            @DataDog/ci-app-libraries
+tests/contrib/unittest_plugin       @DataDog/ci-app-libraries
+ddtrace/ext/ci.py                   @DataDog/ci-app-libraries
+ddtrace/ext/ci_visibility           @DataDog/ci-app-libraries
+ddtrace/ext/test.py                 @DataDog/ci-app-libraries
+ddtrace/internal/ci_visibility      @DataDog/ci-app-libraries
+ddtrace/internal/codeowners.py      @datadog/ci-app-libraries
```

Comment:
```suggestion
ddtrace/internal/codeowners.py      @DataDog/apm-core-python @datadog/ci-app-libraries
```

Reply:
I thought so at first, so I did a search to see what actually uses codeowners in the repo and it seems that it's all CI Vis:
```
romain.komorn@COMP-CLXFH7L2J3 dd-trace-py % rg 'import.*codeowners|from.*codeowners.*import'
tests/internal/test_codeowners.py
1:from ddtrace.internal.codeowners import Codeowners

scripts/check_suitespec_coverage.py
13:from codeowners import Codeowners  # noqa

ddtrace/internal/ci_visibility/recorder.py
236:            from ddtrace.internal.codeowners import Codeowners
```
So maybe the actual change would be for me to add
```
tests/internal/test_codeowners.py.   @datadog/ci-app-libraries
```

Reply:
At the same time, I don't at all mind adding core as a codeowner for that file.